### PR TITLE
Add links to contextualdata directive docs

### PR DESCRIPTION
**Pull request by tsauerwein**
Closes https://github.com/camptocamp/c2cgeoportal/issues/2406

There was already documentation and a good example, so I only added links.

### CODE REVIEW

File: contribs/gmf/src/directives/contextualdata.js [hunk 1]
```diff
@@ -227,7 +229,7 @@ gmf.module.controller('GmfContextualdataController', gmf.ContextualdataControlle
  * Provide a directive responsible of formatting the content of the popover for
  * the contextual data directive.
  *
- * It's main purpose is to configure the template to be used.
+ * Its main purpose is to configure the template to be used.
```
Comment (sbrunner) [inline contribs/gmf/src/directives/contextualdata.js]: Right?

Comment (sbrunner) [inline contribs/gmf/src/directives/contextualdata.js]: OK, thanks :-)

**Comment by tsauerwein** [comment]
Note that I pushed a 2nd commit that fixes the links to example in the API docs (for example here: https://camptocamp.github.io/ngeo/master/apidoc/ngeo.mapDirective.html). JSDoc does not correctly understand relative links when using `@link`. I replaced those links with Markdown links.

**Comment by fredj** [comment]
LGTM, ready to be merged?

**Comment by sbrunner** [comment]
+1

**Comment by tsauerwein** [comment]
Thanks for the reviews.

**Pull request merged**